tests/test_pickling_trampoline.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+import env  # noqa: F401
+
+from pybind11_tests import pickling_trampoline as m
+
+try:
+    import cPickle as pickle  # Use cPickle on Python 2.7
```

Comment:
nit This is an indirect conditional; instead of `try/catch`, can you use `if env.PY2` to make it explicit? (then no comment necessary)